mobile-browser-based-version/src/task_definition/lus_covid.js

```diff
@@ -265,7 +265,7 @@ export const trainingInformation = {
     batchSize: 2,
     // {Object} Compiling information 
     modelCompileData: {
-        optimizer: "adam",
+        optimizer: "rmsprop",
```

Comment:
is this and the deletion of the `threshold` property below intentional? If not, please remove

Reply:
It was not intentional, I just changed it. Thanks for pointing this out!